src/coord/src/catalog.rs

```diff
@@ -112,6 +112,7 @@ pub struct Catalog {
     roles: HashMap<String, Role>,
     storage: Arc<Mutex<storage::Connection>>,
     oid_counter: u32,
+    version: i64,
```

Comment:
Can we call this `revision` instead of `version`? To my ear, `revision` connotes "user-specified changes to catalog" while `version` connotes "system changes to catalog format."